recipes-core/images/nilrt-safemode-initramfs.bb

```diff
@@ -22,7 +22,7 @@ IMAGE_INSTALL_NODEPS += "\
 	${NI_PROPRIETARY_SAFEMODE_PACKAGES} \
 "
 
-BAD_RECOMMENDATIONS:append:pn-${PN} += "shared-mime-info"
+BAD_RECOMMENDATIONS:append:pn-${PN} = " shared-mime-info"
```

Comment:
Check this.

Reply:
From output of `bitbake -e nilrt-safemode-initramfs`, `BAD_RECOMMENDATIONS` contains `shared-mime-info`